dist/index.js

```diff
@@ -0,0 +1,90 @@
+"use strict";
```

Comment:
Let's add `dist` to `.gitignore`

Reply:
Oh I see you're still using this for the tests; I think setting up jest with TS would allow to not rewrite the tests (since it'd pass through babel compile)